common/download-link/index.json-ld

```diff
@@ -0,0 +1,35 @@
+{
+	"@context": {
+		"@version": 1.1,
+		"dct": "http://purl.org/dc/terms/",
+		"title": { "@id": "dct:title", "@container": "@language" },
+		"description": { "@id": "dct:description", "@container": "@language" },
+		"modified": "dct:modified"
+	},
+	"title": {
+		"en": "Download link",
+		"fr": "Lien de téléchargement"
+	},
+	"description": {
+		"en": "Download link with custom styles.",
+		"fr": "Lien de téléchargement avec des styles personalisés."
+	},
+	"modified": "2023-04-20",
+	"componentName": "download-link",
+	"processing": "baseline",
```

Comment:
```suggestion
```
There is no "processing" attribute for this component. That indicator is to know if we need to compile it before any components or if we need to compile it after the components.